### DISCUSSION TITLE
Insígnias e Medalhas 🎖

### DISCUSSION THREAD

**Discussion by ruymon** [diego3g/rsxp-2023#8] [Ideas]
# Insígnias e Medalhas 🎖
### Gamificando o app da RSXP 2023

**🧠 Ideia:** Ter medalhas ou badges para conquistas do pessoal durante toda a experiência do evento. Tal ideia se assemelha muito com o sistema atual na plataforma da @rocketseat.

![image](https://user-images.githubusercontent.com/50504050/228389645-d8775539-daee-4db9-bd33-f107c4328a07.png)

**🔍 Origem da ideia:** Essa ideia é uma adaptação do *insight* que o @diego3g teve durante a [live de kick-off](https://www.youtube.com/watch?v=dDO0YKuHABY). Ele citou a experiência que teve durante um evento, onde recebeu um álbum de figurinhas que foi completando com adesivos que recebeu pelos palestrantes. Com base nisso, pensei em uma maneira de integrar essa ideia de uma maneira digital e que consiga também agregar valor para o App e, por conseguinte, gamificar.

**🗒 Como funciona?** Ao realizar alguma ação, seja ela no evento ou fora dele, o usuário pode ganhar uma medalha e acumular conquistas. Nesse primeiro momento antes do evento, essa função pode estimular venda de ingressos.

**🎖 Ideias de Insígnias:**

- **Early bird |** Dada para usuários da v1 BETA do app.
- **Contributor |** Dada para pessoas que ajudaram no processo de construção do app.
- *Sales related |* Categoria de badges relacionadas à venda de ingressos. Ou seja, reconhecer o usuário que consegue vender `x` quantidade de ingressos.
- *Talk/Keynote related |* Categoria de badges para Talks, ou seja, recebe por comparecer na talk.
- **Hackathon Master! | ** Dada aos vencedores da Hackathon.

Enfim... o céu é o limite para essas badges...

**💰 Como isso agrega valor ao app?** Essa feature estimula o uso do app antes, durante e depois do evento e, cria *hype* para o usuário pegar mais medalhas. Dessa forma promove com que ele curta 100% do evento. Além disso, abre a possibilidade de criar algum tipo de "lembrancinha" com as memórias pós RSXP...

**Comment by claudiolibanez** [comment]
Vamos tentar juntar essa ideia com a ideia da discussão #3 . Seria bem legal

**Comment by ruymon** [comment]
Discussão movida para o #3. :)
